src/containers/PermissionsApplication.jsx

```diff
@@ -1,25 +1,63 @@
 import React from 'react'
 
 import { APPS_DOCTYPE } from 'doctypes'
-import { Q, useQuery, isQueryLoading, hasQueryBeenLoaded } from 'cozy-client'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 import { useI18n } from 'cozy-ui/transpiled/react'
 import Page from 'components/Page'
 import PageTitle from 'components/PageTitle'
 import { withRouter, Link } from 'react-router-dom'
+import {
+  Q,
+  useQuery,
+  isQueryLoading,
+  hasQueryBeenLoaded,
+  models
+} from 'cozy-client'
+
+const {
+  locales: { getBoundT }
+} = models.document
+
+export const completePermission = (key, permission, value) => {
+  const object = {
+    name: key,
+    title: permission
+  }
+  if (value.description) {
+    object.description = value.description
+  }
+  if (value.verbs) {
+    object.verbs = value.verbs
+  }
+  return object
+}
 
 const PermissionsApplication = ({ match }) => {
-  const { t } = useI18n()
+  const { t, lang } = useI18n()
+  const permissionsT = getBoundT(lang)
   const appName = match.params.app
-
   const queryResult = useQuery(
     Q(APPS_DOCTYPE).getById('io.cozy.apps/' + appName),
     {
       as: 'io.cozy.apps/' + appName
     }
   )
 
+  let permissions = []
+  if (queryResult.data && queryResult?.data.length > 0) {
+    const types = Object.entries(queryResult.data[0].attributes.permissions)
+      .map(([key, value]) => {
```

Comment:
```suggestion
      .map(([name, value]) => {
```
and replace the `key` by `name` after that